demisto_sdk/__main__.py

```diff
@@ -207,27 +207,30 @@ def unify(**kwargs):
     'If the --post-commit flag is not supplied: validation will run on all changed files in the current branch, '
     'both committed and not committed. ')
 @click.option(
-    '-p', '--path', help='Path of file to validate specifically, outside of a git directory.'
+    '-p', '--path', help='Path of file to validate specifically, outside of a git directory.', hidden=True
```

Comment:
can't we just remove this? we're not breaking any bc before 1.0

Reply:
No, Itay said to leave it. It's hidden anyway